plenum/server/node.py

```diff
@@ -260,6 +260,7 @@ def __init__(self,
                                notifierEventsEnabled=self.config.SpikeEventsEnabled)
 
         self.replicas = self.create_replicas()
+        self.replicas.set_default_handlers()
```

Comment:
Could you please rename this method to something like "register_monitor_handler" or choose different name that shows that monitor is being affected not replicas.

Reply:
Renamed